week7/answers.sql

```diff
@@ -1,3 +1,284 @@
 -- Create your tables, views, functions and procedures here!
 CREATE SCHEMA destruction;
 USE destruction;
+
+CREATE TABLE players (
+  player_id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT,
+  first_name VARCHAR(30),
```

Comment:
Comment througout: be sure to include NOT NULL (or you might find some math breaking if someone were to insert a NULL value).

Reply:
NOT NULL has been added,